packages/slackctl/README.md

```diff
@@ -0,0 +1,172 @@
+# slackctl
+
+A utility for managing Slack [App](https://api.slack.com/start/overview) configurations as code with helpful integrations to make dev environments easy.
+
+⚠️ Slack doesn't currently support this. To make this project work, we rely heavily on undocumented apis and client tokens. These tokens should be protected very carefully. Use at your own risk. This is not intended to be used for production, though it does make dev a whole lot easier.
+
+# Table of Contents
+
+- [Install](#install)
+- [Updating a Slack App](#updating-a-slack-app)
+- [App Configuration File](#app-configuration-file)
+  - [Setting up a `local` config](#setting-up-a-local-config)
+- [Getting a session token](#getting-a-session-token)
+
+## Install
+
+Install the package globally for usage as a cli.
```

Comment:
seems like maybe a good candidate for `npx`? then people could install locally easily also. I always dislike "requiring" global installs where avoidable.